Docs for Organizations Phase I (#2063)
neondatabase/website: content/docs/manage/accounts.md

---
title: Accounts
subtitle: Find out which account type is right for you
enableTableOfContents: true
---

Neon offers two account types for you to choose from:

- **Personal account**
- **Organization account** (Early Access)

## Personal accounts

> When you first sign up with Neon, whether you're signing up on your own or you're invited to join, you start with a personal account. During your [onboarding](/docs/get-started-with-neon#step-2---onboarding-in-the-neon-console)), you are prompted to create your first personal project that you can get started with.

Reviewer: ```suggestion
When you first sign up with Neon, whether you're signing up on your own or you're invited to join, you start with a personal account. During your [onboarding](/docs/get-started-with-neon/signing-up#step-2-onboarding-in-the-neon-console), you are prompted to create your first personal project that you can get started with.
```

You can always access your personal account settings from your **Profile** dropdown.

![personal account settings](/docs/manage/personal_account.png)

If you want to work with other people on this project, your options are:

- [Share](/docs/guides/project-sharing-guide) the project with another Neon user
- [Transfer](/docs/manage/orgs-project-transfer) the project from a personal to an Organization-level project, where you can then invite other users

## Organization accounts

<EarlyAccess/>

> Any Neon user in our Early Access Program can [create an Organization account](/docs/manage/organizations#create-an-organization), which allows for sharing projects across the various members that make up your team. Whether you create an organization or are invited to join one, you still retain your personal account, letting you manage personal projects independently of any organizations you belong to.

Reviewer: ```suggestion
Any Neon user in our Early Access Program can [create an Organization account](/docs/manage/organizations#create-an-organization), which allows you to share projects with your team members. Whether you create an organization or are invited to join one, you still retain your personal account, letting you manage personal projects independently of any organizations you belong to.
```
